<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>eventViewer</class>
 <widget class="QDialog" name="eventViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EventViewer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/res.qrc">
    <normaloff>:/res/image/icon/flashlight.png</normaloff>:/res/image/icon/flashlight.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="icon">
       <iconset resource="../res/res.qrc">
        <normaloff>:/res/image/icon/flashlight.png</normaloff>:/res/image/icon/flashlight.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Event</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QToolButton" name="btnEventExport">
             <property name="text">
              <string>Export</string>
             </property>
             <property name="icon">
              <iconset resource="../res/res.qrc">
               <normaloff>:/res/image/icon/219.png</normaloff>:/res/image/icon/219.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="btnEventLoad">
             <property name="text">
              <string>Load</string>
             </property>
             <property name="icon">
              <iconset resource="../res/res.qrc">
               <normaloff>:/res/image/icon/218.png</normaloff>:/res/image/icon/218.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="btnEventAdd">
             <property name="text">
              <string>+</string>
             </property>
             <property name="icon">
              <iconset resource="../res/res.qrc">
               <normaloff>:/res/image/icon/addition.png</normaloff>:/res/image/icon/addition.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="btnEventDel">
             <property name="text">
              <string>-</string>
             </property>
             <property name="icon">
              <iconset resource="../res/res.qrc">
               <normaloff>:/res/image/icon/delete.png</normaloff>:/res/image/icon/delete.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="eventView">
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="icon">
       <iconset resource="../res/res.qrc">
        <normaloff>:/res/image/icon/browse.png</normaloff>:/res/image/icon/browse.png</iconset>
      </attribute>
      <attribute name="title">
       <string>EventView</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QToolButton" name="btnExport">
             <property name="text">
              <string>Export</string>
             </property>
             <property name="icon">
              <iconset resource="../res/res.qrc">
               <normaloff>:/res/image/icon/219.png</normaloff>:/res/image/icon/219.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="btnClear">
             <property name="text">
              <string>Clear</string>
             </property>
             <property name="icon">
              <iconset resource="../res/res.qrc">
               <normaloff>:/res/image/icon/trash.png</normaloff>:/res/image/icon/trash.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Cache Size</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox">
             <property name="minimum">
              <number>16</number>
             </property>
             <property name="maximum">
              <number>4096</number>
             </property>
             <property name="value">
              <number>64</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="chkView">
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="text">
              <string>View</string>
             </property>
             <property name="icon">
              <iconset resource="../res/res.qrc">
               <normaloff>:/res/image/icon/browse.png</normaloff>:/res/image/icon/browse.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget">
           <property name="modelColumn">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnClear</sender>
   <signal>clicked()</signal>
   <receiver>listWidget</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>8</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
